types/react-codemirror/index.d.ts

```diff
@@ -1,6 +1,7 @@
 // Type definitions for React Codemirror v0.2.6
```

Comment:
your type change isn't in 0.2.6, it's in 1.0.0. 1.0.0 introduces other changes as well - you should update the typings to match all of 1.0.0's changes if you're going to bump the version.
See https://github.com/JedWatson/react-codemirror/blob/master/HISTORY.md#v100--2017-05-14